Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "IndirectArrays"
+uuid = "9b13fd28-a010-5f03-acff-a1bbcff69959"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+
+[compat]
+Compat = "≥ 0.53.0"
```

Comment:
All of these need to have upper bounds in order to be efficiently merged to the registry: https://discourse.julialang.org/t/announcement-automatic-merging-for-the-general-registry/29961.

Reply:
i've removed the inequalities, and added a CompatHelper commit to help keep the compat section up-to-date going forward.  not sure if that commit is all that's needed to install it though.